platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.mita

```diff
@@ -0,0 +1,33 @@
+/********************************************************************************
+ * Copyright (c) 2017, 2018 Bosch Connected Devices and Solutions GmbH.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * Contributors:
+ *    Bosch Connected Devices and Solutions GmbH - initial contribution
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+
+package platforms.xdk110;
+
+import platforms.xdk110;
+
+/**
+ * @brief Enables the power control for the extension board such that the extension bus gets power supply, also put
+ *        all the GPIO pins of the extension port pins into disabled state to save power
+ *
+ * @retval 0 in the case of success or an error code otherwise.
+ *
+ * @note In reality this function returns a Retcode_T. However since it doesn't accept any arguments we can't declare it as checked (for now). 
+ * 		 You should therefore check its return type yourself like this:
+ * @code
+ * let err = BSP_ExtensionPort_Connect();
+ * if(err != 0) {
+ *     throw Exception;
+ * }
+ * @endcode
+ */
+export native unchecked fn BSP_ExtensionPort_Connect(): int32 header "BSP_ExtensionPort.h";
```

Comment:
Not sure if this what we want ... should controlling the extension bus not rather be part of a resource configuration?

Reply:
Well, in some cases you just want to provide power to something external. We could make some kind of `setup externalPower { enable2v5 = true; }` though.